.gitignore

```diff
@@ -10,8 +10,9 @@ ctaocrypt/src/src/
 *.cache
 .dirstamp
 *.user
-configure
-config.*
+configure
+config.log

```

Comment:
Please revert the `config.*` change. There are several files that get generated that need to be ignored. Like `config.h` and `config.in`. Thanks

Reply:
I reverted them, but now you won't notice the generated files, and you can always ignore them using `.git/info/exclude`.